### ISSUE TITLE
Creating a subset list of MONDO terms which are relevant to drug repurposing.

### ISSUE THREAD

**Issue by DnlRKorn** [monarch-initiative/mondo#6262]
This is following up on a conversation at the curation call on May 11th about the EveryCure ranking project. We're trying to query translator TRAPI endpoints looking for ranks for drug-disease pairs. Presently we are working by iterating through all possible MONDO terms as a rough approximation of all possible diseases what we could target, but obviously that can be pruned down much further. 

Briefly summarizing our conversation, we talked about removing susceptibility subtypes from the pool of MONDO terms, and removing terms which are above a certain number of hops from leaf nodes or have specific node properties.

**Comment by nicolevasilevsky** [comment]
@DnlRKorn to be clear, is what you want a list of Mondo disease terms with the susceptibility terms excluded? If anything else is needed, can you please provide a detailed list with the criteria you need for this subset?

Feel free to join our bi-weekly tech calls on Fridays at 9am PT (next call is on May 28)

Thank you!

**Comment by DnlRKorn** [comment]
@nicolevasilevsky I believe we're interested in removing susceptibility terms and "terms which are unlikely to be a useful diagnostic criteria". This second one is complicated, what I've tried to look at and discuss is how high up the tree is useful for a reasonable disease; the example I have refered before is Schizophrenia 7 (MONDO:0011295) versus Schizophrenia (MONDO:0005090). Clearly we'd want our list to include both, but MONDO:0005090 isn't a leaf node, so that isn't a very useful criteria...

**Comment by matentzn** [comment]
@DnlRKorn - thanks for your interest! We are very keen to help you with your use case, but we are a bit snowed under right now with finishing the Mondo paper; to help you more, I think we need to talk in person. Would it be possible to postpone the discussion on this to May 28, and you joining the call? Until then, maybe you can develop a more operational notion of the "terms which are unlikely to be a useful diagnostic criteria", and we can advice on how this is best implemented.

**Comment by nicolevasilevsky** [comment]
@DnlRKorn would you be able to join one of our future calls?

**Comment by DnlRKorn** [comment]
Yes! Sorry it's been very busy since the relay, I'll try and make time this
week!

**Comment by nicolevasilevsky** [comment]
Use case/Exact pipeline this will be applied to:

- wants a list of diseases that are drug treatable
- terms like Mendelian disease are not treatable per se
-  would like to prune out diseases like this to avoid unnecessary calculations of disease targets
- want to be able to find potential drug-disease pairs
- from a user query perspective - users could be querying for specific diseases, then having extra diseases in the hierarchy won't be as problematic
- nice to see the shape of the ontology, 80% of the ontology is leaf nodes. Removing grouping classes would probably only remove 10% of the classes
- Daniel would like to say if he finds a hit, would like to say it could be interesting for a physician
- the border b/w what is a grouping class and what isn't is fuzzy
- rather than filtering certain classes, may consider information content metric for diseases. Eg Mendelian disease has a low information content, therefore it would be less relevant to report
- can use ontology to report IC
- run a semantic similarity analysis, the higher the term is in the hierarchy, the lower the IC and then they could be valued less in the analysis
- we need a way to compare the Mondo diseases with a real source of diagnoses- ie if you had  a database that contains true diagnoses 
- ICD is used in clinical diagnosis, everything with a mapped ICD10 code (that is lower in their coding system - must have a decimal to indicate it is a billable code
- need to improve the ICD10 code mappings in Mondo
- note, any measure that you apply to diseases will be a lossy
- Nico recommends trying to make the IC score a bit more meaningful, like by considering gene relationships - like if you have a lot of associated genes, this could change the weight of the IC score
- Curation/technical team should think further about how to develop a product/subset that approximates diagnosability in the IC score
- Sanya is indexing PubMed with Mondo terms - this could be another approach to identifying the Mondo terms that are potentially diagnosable terms
- combine ontological IC score and PMIDs and matched Mondo terms - this could be a good approach
